tests/modules/nf-scil/reconst/diffusivitypriors/main.nf

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { RECONST_DIFFUSIVITYPRIORS as MEAN_PRIORS} from '../../../../../modules/nf-scil/reconst/diffusivitypriors/main.nf'
+include { RECONST_DIFFUSIVITYPRIORS as COMPUTE_PRIORS} from '../../../../../modules/nf-scil/reconst/diffusivitypriors/main.nf'
+
+workflow test_reconst_diffusivitypriors_compute_priors {
+    
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['reconst']['diffusivitypriors']['fa'], checkIfExists: true),
+        file(params.test_data['reconst']['diffusivitypriors']['md'], checkIfExists: true),
+        file(params.test_data['reconst']['diffusivitypriors']['ad'], checkIfExists: true),
+        []
+    ]
+
+    COMPUTE_PRIORS ( input )
+}
+
+workflow test_reconst_diffusivitypriors_mean_priors {
+
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['reconst']['diffusivitypriors']['fa'], checkIfExists: true),
+        file(params.test_data['reconst']['diffusivitypriors']['md'], checkIfExists: true),
+        file(params.test_data['reconst']['diffusivitypriors']['ad'], checkIfExists: true),
```

Comment:
Here, I think the test works without those files supplied. If so, replace them with `[]`.

Reply:
In fact it doesn't work